aim/sdk/adapters/pytorch_ignite.py

```diff
@@ -0,0 +1,201 @@
+from typing import Any, Callable, Dict, List, Optional, Union
+
+from aim import Run
+from aim.ext.resource.configs import DEFAULT_SYSTEM_TRACKING_INT
+
+try:
+    from torch.optim import Optimizer
+except ImportError:
+    raise RuntimeError(
+        'This contrib module requires PyTorch to be installed. '
+        'Please install it with command: \n pip install torch'
+    )
+try:
+    from ignite.contrib.handlers.base_logger import BaseLogger, BaseOptimizerParamsHandler, BaseOutputHandler
+    from ignite.engine import Engine, Events
+except ImportError:
+    raise RuntimeError(
+        'This contrib module requires PyTorch Ignite to be installed. '
+        'Please install it with command: \n pip install pytorch-ignite'
+    )
+
+
+class AimLogger(BaseLogger):
+    """
+        `Aim` tracking client handler to log parameters and metrics during the training
+        and validation.
+
+        Args:
+            repo: Aim repository path
+            experiment: Aim run's experiment name
+            train_metric_prefix: prefix to identify training metrics
+            val_metric_prefix: prefix to identify validation metrics
+            test_metric_prefix: prefix to identify test metrics
+            system_tracking_interval: Sets the tracking interval in seconds
+                for system usage metrics (CPU, Memory, etc.)
+    """
+    def __init__(self,
+                 repo: Optional[str] = None,
+                 experiment: Optional[str] = None,
+                 train_metric_prefix: Optional[str] = 'train_',
+                 val_metric_prefix: Optional[str] = 'val_',
+                 test_metric_prefix: Optional[str] = 'test_',
+                 system_tracking_interval: Optional[int]
```

Comment:
can we add `log_system_params` argument here as well? the rest of `Run` arguments are configurable